kiosk.py

```diff
@@ -15,13 +15,27 @@
 # Copyright (c) 2015, Serapheim Dimitropoulos
 #
 
-from flask import Flask, render_template
+from flask import Flask, render_template, request
 
 app = Flask(__name__)
 
+
 @app.route('/')
 def hello():
     return render_template('hello.html')
 
+
+@app.route('/login', methods=['GET', 'POST'])
+def login():
```

Comment:
Could you change that to @app.route('/signin', ...etc) and def signin(): pretty please? :P